libogc/lwp_heap.inl

```diff
@@ -1,6 +1,8 @@
 #ifndef __LWP_HEAP_INL__
 #define __LWP_HEAP_INL__
 
+#include "lwp_heap.h"
```

Comment:
why is the include of the header here, and not where the inl is included? (like in [lwp_heap.c](https://github.com/devkitPro/libogc/pull/138/files#diff-cd8dda5bd864cfde73409b8ea8b2babccbe5ab5bba93cddd7f338f8ec397639a), it should keep the include of lwp_heap.h, but also include this inline.

this applies for all includes of these inline functions